tests/integration/Owncloud/OcisSdkPhp/OcisTest.php

```diff
@@ -134,4 +134,13 @@ public function testCreateDriveInvalidQuota(int $quota): void
         // no new drive should have been created
         $this->assertCount($countDrivesAtStart, $ocis->listMyDrives());
     }
+
+    public function testListGroups(): void
```

Comment:
create more tests
1. every parameter
2. user that has no permissions to read groups
3. unit tests for error handling